### ISSUE TITLE
Unable to restore arch id mappings for 'm88k' (likely caused by changes in the mapped view schema)

### ISSUE THREAD

**Issue by xusheng6** [Vector35/binaryninja-api#5718]
A user reported that opening a previously saved database causes the following warning messages:

```
[BinaryView] Updated load schema detected for BinaryViewType: 'Mapped'. Old load schema available at: 'old:Mapped'
[Default] Unable to restore arch id mappings for 'm88k'
[Default] Unable to restore arch id mappings for 'm88k'
```

The database involves a custom architecture plugin developed by the user

**Comment by xusheng6** [comment]
V35 folks should search for `nookie slakeless crumbled salarying wert` to find the relevant arch plugin code and database

**Comment by plafosse** [comment]
Fixed in 4.1.5850

**Comment by pmackinlay** [comment]
Confirming fixed in 4.2.5850-dev, thanks.
